Header header
int8 speed_l
int8 speed_r
int8 volt
int8 current_l
int8 current_r
int8 error
int8 acceleration
int8 mode
int8 regulator
int8 timeout

